views/container/data-actions.php

```diff
@@ -0,0 +1,9 @@
+<?php
+defined( 'ABSPATH' ) || die( 'Cheatin’ uh?' );
+
+?>
+<div class="imagify-data-actions-container" data-id="<?php echo (int) $data['media_id']; ?>" data-context="<?php echo esc_attr( $data['context'] ); ?>">
+	<?php echo $data['content']; ?>
```

Comment:
no need for sanitize either?

Reply:
This is a container for the above buttons, so it contains HTML tags from our templates. `wp_kses()` feels overkill here.